views/base.pug

```diff
@@ -30,10 +30,13 @@ html(lang=getLocale() === 'fr' ? 'fr' : 'en')
       header
         include _includes/phaseBanner
         include _includes/fip
+      
       main
         if errors
           include _includes/errorList.pug
 
+        include _includes/backButton
```

Comment:
👌

Reply:
Cool, but let's put it above the errors.